tajo-core/src/main/java/org/apache/tajo/engine/planner/global/ExecutionBlock.java

```diff
@@ -120,25 +171,49 @@ public boolean hasUnion() {
     return hasUnionPlan;
   }
 
-  public void addBroadcastTable(String tableName) {
-    broadcasted.add(tableName);
-    enforcer.addBroadcast(tableName);
+  public boolean isUnionOnly() {
+    return isUnionOnly;
   }
 
-  public void removeBroadcastTable(String tableName) {
-    broadcasted.remove(tableName);
-    enforcer.removeBroadcast(tableName);
+  public void addBroadcastRelation(ScanNode relationNode) {
```

Comment:
I used the term 'broadcastable'. If this term is used with 'table' together, i.e., broadcastableTable, it makes readers confused.